src/main/java/com/whichclasses/scraper/page/DeptListPage.java

```diff
@@ -60,8 +59,24 @@ private DeptListModel getModel() {
    * @return
    */
   private String normalizeDepartmentId(String name) {
-    return name.replaceAll("[\\s_]", "");
+    return name.replaceAll("[\\s]", "");
+    //return name.replaceAll("[\\s_]", "");
```

Comment:
TCE website is stupid, for some reason it uses underscores in the 'crssub' query field for the department list links, but doesn't use them in the course list links.

E.g.:
Department List:
<a href="GenerateReport.aspx?Report=DEPTCOURSE&amp;&amp;crssub=C_SC&amp;&amp;deptnm=Computer+Science+(C+SC)">Computer Science (C SC)</a>

Course:
<a href="GenerateReport.aspx?Report=DEPTONECLASS&amp;&amp;crstitle=Introduction+to+Computer+Science&amp;crssub=CSC&amp;crsnum=127A">CSC 127A</a>